default/AI/ProductionAI.py

```diff
@@ -1640,6 +1640,29 @@ def generateProductionOrders():
                 else:
                     print "Error failed enqueueing %s at planet %d (%s) , with result %d"%(bldName, pid, planet.name, res)
 
+
+    queued_clny_bld_locs = [element.locationID for element in productionQueue if element.name.startswith('BLD_COL_')]
+    colony_bldg_entries = ([entry for entry in foAI.foAIstate.colonisablePlanetIDs.items() if entry[1][0] > 60 and
+                           entry[0] not in queued_clny_bld_locs and entry[0] in ColonisationAI.empire_outpost_ids]
+                           [:PriorityAI.allottedColonyTargets+2])
+    for entry in colony_bldg_entries:
+        pid = entry[0]
+        bldName = "BLD_COL_"+entry[1][1][3:]
```

Comment:
My menu was ```Settings->Editor->Code Style->Python``` (and for some reason it took me a while to realize I had to double-click on "Code Style" to get it to list the language options), and that did the trick, thanks.  I disabled the automatic spaces around bitwise, additive and multiplicative operators.  I'll take care of those manually when they seem appropriate.